### PR TITLE
AM-2717 MFA enrolment views fixes

### PR DESCRIPTION
**Pull request by lgw-gravitee**
## :id: Reference related issue. 
https://gravitee.atlassian.net/browse/AM-2717

## new view

image

### CODE REVIEW

**Review by mstasgravitee — APPROVED** [review]

File: gravitee-am-ui/src/app/domain/applications/application/advanced/factors/info-banner/info-banner.component.html [hunk 1]
```diff
@@ -16,11 +16,15 @@
...
 -->
 
-<div *ngIf="showInfo" fxLayout="row" fxLayoutAlign="center center" class="show-info" [style]="getColors()">
-  <div fxFlex="5" style="padding: 10px"><mat-icon>info_outline</mat-icon></div>
-  <div fxFlex="65" style="gap: 4px">{{ text }}</div>
-  <button fxFlex="25" (click)="gotoLink($event)" mat-raised-button class="more-info-button">
-    {{ buttonName }}
-  </button>
-  <div fxFlex="5" fxLayoutAlign="end center" class="close-info"><mat-icon (click)="closeInfo()">close</mat-icon></div>
+<div *ngIf="showInfo" fxLayout="row" fxLayoutAlign="start center" class="show-info" [style]="getColors()">
```
Comment (mstasgravitee) [inline gravitee-am-ui/src/app/domain/applications/application/advanced/factors/info-banner/info-banner.component.html:19]: We could add here min-width: 320px; to keep the background on all elements in smaller screens. What do you think?

File: gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/conditional/mfa-conditional.component.html [hunk 2]
```diff
@@ -36,32 +36,29 @@
...
         [(ngModel)]="this.enrollment.enrollmentSkipRule"
         (keyup)="updateSkipRule($event)"
       />
-      <mat-hint>
-        <a href="https://documentation.gravitee.io/apim/guides/gravitee-expression-language" target="_blank">
-          <p class="link">Read more about Gravitee Expression Language</p>
-        </a>
-      </mat-hint>
     </mat-form-field>
+    <a class="link" href="https://documentation.gravitee.io/apim/guides/gravitee-expression-language" target="_blank">
```
Comment (mstasgravitee) [inline gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/conditional/mfa-conditional.component.html:57]: please update this also in other components for consistency 
image

**Review by mstasgravitee — CHANGES_REQUESTED** [review]

File: gravitee-am-ui/src/app/domain/applications/application/advanced/factors/info-banner/info-banner.component.html (hunk 1 above)
Comment (lgw-gravitee) [reply gravitee-am-ui/src/app/domain/applications/application/advanced/factors/info-banner/info-banner.component.html:19]: but it doesnt change anything. On smaller devices than 320px the view is entirely not usable

File: gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/conditional/mfa-conditional.component.html (hunk 2 above)
Comment (lgw-gravitee) [reply gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/conditional/mfa-conditional.component.html:57]: you probably didnt update the branch. its already updated.

**Review by mstasgravitee — APPROVED** [review]

**Pull request merged**